test/ecto/adapters/clickhouse/connection_test.exs

```diff
@@ -750,7 +836,7 @@ defmodule Ecto.Adapters.ClickHouse.ConnectionTest do
       |> select([r], r.x)
       |> where([], fragment(~s|? = "query\\?"|, ^10))
 
-    assert all(query) == ~s[SELECT s0."x" FROM "schema" AS s0 WHERE ({$0:Int64} = "query?")]
+    assert all(query) == ~s[SELECT s0."x" FROM "schema" AS s0 WHERE {$0:Int64} = "query?"]
```

Comment:
Not sure about this one. In this case a "binary" equals(x, y) op is not wrapped since it's inside a fragment.

Reply:
Moved to https://github.com/plausible/ecto_ch/pull/209